modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerSelfTest.java

```diff
@@ -423,6 +424,21 @@ void bulkUpdateIncrementsVersionByOne() {
     @Test
     void bulkUpdateDoesntIncrementVersionInCaseOfError() {
         int versionBefore = manager.latestCatalogVersion();
+        tryApplyAndCheckExpect(
+                        List.of(TestCommand.empty(), TestCommand.ok(), TestCommand.empty(), TestCommand.ok(),  TestCommand.empty()),
+                        false, true, false, true, false
+        );
+
+        Catalog updatedCatalog = manager.catalog(manager.latestCatalogVersion());
+
+        assertNotNull(updatedCatalog);
+
+        assertEquals(4 + versionBefore, updatedCatalog.objectIdGenState());
```

Comment:
How does `objectIdGenState` relates to `versionBefore`?
I think this test should be reworked

1. save `catalog before`
2. apply update
3. check catalogVersionBefore incremented by `1`
4. check objectIdGenState incremented by `2`

Reply:
reworked  these tests